### PR TITLE
(chore) Bump turbo

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Bumps [turbo](https://github.com/vercel/turbo) to the latest available version, enabling the following improvements:

- https://turbo.build/blog/turbo-1-11-0
- https://turbo.build/blog/turbo-1-12-0
- https://turbo.build/blog/turbo-1-13-0

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
         run: yarn verify
 
       - name: Run build
-        run: yarn turbo build --color --concurrency=5
+        run: yarn turbo build --concurrency=5
```
Comment (denniskigen) [inline .github/workflows/ci.yml:52]: The `color` flag is now included by default in all terminal output.

File: .github/workflows/size.yml [hunk 2]
```diff
@@ -26,4 +26,4 @@ jobs:
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           minimum-change-threshold: 10000 # 10 KB
-          build-script: "turbo run build --color"
+          build-script: "turbo build"
```
Comment (denniskigen) [inline .github/workflows/size.yml:29]: We don't need to type the `run` command either.

File: turbo.json [hunk 3]
```diff
@@ -4,10 +4,7 @@
     "build": {
       "outputs": ["dist/**"]
     },
-    "test": {
-      "dependsOn": ["build"],
-      "inputs": ["src/**/*.tsx", "src/**/*.ts", "test/**/*.ts", "test/**/*.tsx"]
-    },
+    "test": {},
```
Comment (denniskigen) [inline turbo.json:7]: `test` shouldn't have a topological dependency on `build`. Also the inputs aren't necessary AFAICT.

File: package.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "homepage": "https://github.com/openmrs/openmrs-form-engine-lib#readme",
   "scripts": {
-    "lint": "TIMING=1 eslint src --ext .ts,.tsx --fix",
```
Comment (denniskigen) [inline package.json:10]: We no longer need this `TIMING` environment variable as all output to `stdout` gets cached by turbo.

**Review by samuelmale — APPROVED** [review]

**Pull request merged**